src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const bodyElement = document.body;
+
+const dataSalary = [...bodyElement.querySelectorAll('li[data-salary]')];
+const listPeople = [...bodyElement.querySelectorAll('li')];
+
+const salaries = dataSalary
+  .map((elem) => +elem.getAttribute('data-salary').replace(/[$,]+/g, ''))
+  .sort((a, b) => b - a);
+
+const sortedArray = [];
+
+function sortList(list) {
+  for (let i = 0; i < list.length; i++) {
+    const salary = salaries[i];
+    const stringSalary = salary
+      .toLocaleString('en-us')
+      .padStart(salary.toLocaleString('en-us').length + 1, '$');
+
+    const person = list.filter(
+      (elem) => elem.getAttribute('data-salary') === stringSalary,
+    );
+
+    person.forEach((elem) => {
+      if (!sortedArray.includes(elem.textContent)) {
+        sortedArray.push(elem.textContent);
+      }
+    });
+  }
+}
+
+function getEmployees(list) {
+  for (let i = 0; i < list.length; i++) {
+    list[i].textContent = sortedArray[i];
```

Comment:
This line assumes that `sortedArray` has the same length as `list`. If there are duplicate salaries, `sortedArray` might be shorter, leading to undefined values being assigned to `list[i].textContent`. Ensure that `sortedArray` and `list` are synchronized in length.